### ISSUE TITLE
Tarrant County Records

### ISSUE THREAD

**Issue by DMalone87** [TubmanProject/data_scraper#8]
# Overview
Data available only via a search tool on the site. You can get bulk data only by doing a search by date. Not sure if this is real time or how often the data is updated. Data is displayed on the page which makes parsing kind of a headache. There's a very low quantity of records and only a few fields are available.

There is another possible avenue for access if you apply for their web-based access service. This may be worth exploring with some help from TCJC, Internet Bar, or another legal organization if we can get free access.

TL;DR - Pain in the ass... proceed with caution.

# Resources Found
**Subscriber Access to Court Dockets**
Grants web-based access to court docket information. Requires a monthly payment and a Bar ID.
https://egov.tarrantcounty.com/web/guest/departments/district-clerk/subscriber-access

**Court Records Search**
Manually search for court records. You are able to select for Criminal cases and search by date range to get bulk results. Results are returned in HTML only.
https://odyssey.tarrantcounty.com/publicaccess/
